rs/replicated_state/src/page_map.rs

```diff
@@ -444,18 +447,21 @@ pub struct PageMap {
     /// The map containing pages overriding pages from `storage`.
     /// We need these pages to be able to reconstruct the full heap.
     /// It is reset when `strip_all_deltas()` method is called.
+    #[validate_eq(Ignore)]
     page_delta: PageDelta,
 
     /// The map containing deltas accumulated since the last flush to disk.
     /// It is reset when `strip_unflushed_delta()` or `strip_all_deltas()` methods are called.
     ///
     /// Invariant: unflushed_delta ⊆ page_delta
+    #[validate_eq(Ignore)]
     unflushed_delta: PageDelta,
 
     has_stripped_unflushed_deltas: bool,
```

Comment:
Actually, is this even going to work? If you compare the `PageMap` from the loaded state with the previous one, they will differ on `base_height`, and `has_stripped_unflushed_deltas` is also never reset to `false`. Or is the idea to only compare after we did `switch_to_checkpoint`?

Reply:
Yes, because we are not calling it yet :-) Let me run the benchmark and see how it goes. This one should fail, I also want to check what else

Reply:
So, the height is the same, the `has_stripped_unflushed_delta` diverges (the only one). Validation takes ca 100ms for 80k canisters

Reply:
Hmm, but why do they have the same height? And which height is it?

Reply:
I see we change the base height in `switch_to_checkpoint` [here](https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/replicated_state/src/page_map.rs?L880). Does that mean you are comparing after that point?

Reply:
Yes, in my benchmark I was comparing after the switch.